handbook/company.md

```diff
@@ -53,152 +53,152 @@ Empathy leads to better understanding, better communication, and better decision
+  - As time allows  ("I would have written a shorter letter, but I did not have the time." -Blaise Pascal).
+- Make time for self-care.
+  - This helps you bring your best self when communicating with others, making decisions, etc.
+  - Consider taking a break or going for a walk.
+  - Take time off (it is better to have 100% focus for 80% of the time than it is to have 80% focus for 100% of the time).
+  - Think about how to best organize your day/work hours to fit your life and maximize your focus.
+- Better focus.
 
 
 ### 🔵 Objectivity
 <!-- TODO: write short preamble, like the others --> 
 
-- be curious
-  - ask great questions & take the time to truly listen
-  - listen intently to feedback, and genuinely try to understand (especially constructive criticism)  <!-- TODO: Trust the feedback from counterparts. It’s easy to quickly say “no” or ignore feedback because we’re busy and we often default to our way of thinking is right. Trust that your counterpart is making a good suggestion and give it the time/consideration it deserves. -->
-  - see failure as a beginning (it is rare to get things right the first time)
-  - question yourself ("why do I think this?")
-- underpromise, overdeliver
-  - quality results often take longer than we anticipate
-  - be practical about your limits, and about what's possible with the time and resources we have
-  - be thorough (don't settle for "the happy path"; every real-world edge case deserves handling)
-- prioritize truth (reality)
-  - be wrong, show your work (it's better to make the right decision than it is to be right)
-  - "strong opinions, loosely held"  (proceed boldly, but change your mind in the face of new evidence)
-  - avoid sunk cost fallacy (getting attached to something just because you invested time working on it, or came up with it)
-  - be fair to competitors ("may the best product win.")
-  - give credit where credit is due; don't show favoritism <!-- as it breeds resentment, destroys employee morale, and creates disincentives for good performance.  Seek out ways to be fair to everyone - https://about.gitlab.com/handbook/values/#permission-to-play -->
-  - facts, over commentary
-- speak computer to computers
-  - a lucky fix without understanding does more harm than good
-  - when something isn't working, use the scientific method
-  - especially when there is a bug, or when something is slow, or when a customer is having a problem
-  - assume it's your fault
-  - assume nothing else
-- better rigour
+- Be curious.
+  - Ask great questions & take the time to listen genuinely. 
+  - Listen intently to feedback, and truly try to understand (especially constructive criticism).  <!-- TODO: Trust the feedback from counterparts. It’s easy to quickly say “no” or ignore feedback because we’re busy and we often default to our way of thinking is right. Trust that your counterpart is making a good suggestion and give it the time/consideration it deserves. -->
```

Comment:
```suggestion
  - Listen intently to feedback and genuinely try to understand (especially constructive criticism).  <!-- TODO: Trust the feedback from counterparts. It’s easy to quickly say “no” or ignore feedback because we’re busy and we often default to our way of thinking is right. Trust that your counterpart is making a good suggestion and give it the time/consideration it deserves. -->
```

Reply:
...So, I'll switch this back to "genuinely" to avoid repetition. And Grammarly told me to remove the comma, so I'll abide 😄